parts/ic/transceiver/can/MCP2562-ESN.json

```diff
@@ -18,11 +18,10 @@
         false,
         "Microchip"
     ],
-    "model": "00000000-0000-0000-0000-000000000000",
+    "model": "b99785a3-c0da-4658-969d-6287cec3be05",
     "orderable_MPNs": {
-        "340cc66d-a0db-4e3a-8c2a-3fa498baf856": "MCP2562-H/SN",
         "ab7348d5-75d4-46f1-ba3a-e1b1313b1479": "MCP2562T-E/SN",
-        "abcc3ee9-a73c-4e51-8b4c-3547651109f7": "MCP2562-H/SN"
```

Comment:
I didn't notice the duplicate before, but I think `MCP2562-H/SN` is a valid orderable number number (high-temperature).

Reply:
Yep, you are pretty fast with your reviews ;) If I understood the orderable MPNs correctly, they should just differ in packaging. The temperature range is a property of the device it-self. Or am I wrong?

Reply:
Hmmm, I think you're right. I looked at the convention, and it even uses temperate range as an example:
> Use inheritance for only slightly differing parts (like different packaging, temperature range, etc.).